src/index.css

```diff
@@ -0,0 +1,198 @@
+/* green: #677767 */
+/* beige: #F7F6F1 */
+/* white: #fff */
+/* black: #000 */
+
+html, body {
+    font-size: calc(14px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+}
+
+body {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 15px;
+    font-weight: 300;
+
+    min-width: 320px;
+    line-height: 1.2;
+    background-color: #F7F6F1;
+
+    display: flex;
+    align-items: center;
+}
+
+body::-webkit-scrollbar-track {
+    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+    border-radius: 10px;
+}
+
+body::-webkit-scrollbar {
+    width: 10px;
+}
+
+body::-webkit-scrollbar-thumb {
+    border-radius: 10px;
+    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, .3);
+    background-color: #E4E3DE;
+}
+
+#dashboard,
+.dashboard__heading,
+.dashboard__card-heading {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#authorization {
+    display: none;
+}
```

Comment:
And add `margin: 0 auto`.

Reply:
This is for the problem with centering the form.

Reply:
Done.